pack/eoe/eoec.json

```diff
@@ -0,0 +1,869 @@
+        "name": "Ellsworth's Boots",
+        "pack_code": "eoec",
+        "position": 234,
+        "quantity": 1,
+        "skill_agility": 1,
+        "skill_wild": 1,
+        "text": "Limit 1 [[Footwear]] per investigator.\n[reaction] After the last clue is discovered from your location, exhaust Ellsworth's Boots: Move to a connecting location, or discard Ellsworth's Boots to move to any revealed location.",
+        "traits": "Item. Clothing. Footwear.",
+        "type_code": "asset",
+        "xp": null
+    },
+    {
+        "code": "08735",
+        "cost": 2,
+        "deck_limit": 1,
+        "encounter_code": "memorials_of_the_lost",
+        "encounter_position": 6,
+        "faction_code": "neutral",
+        "illustrator": "Drazenka Kimpel",
+        "name": "Kensler's Log",
+        "pack_code": "eoec",
+        "position": 235,
+        "quantity": 1,
+        "skill_intellect": 1,
+        "skill_wild": 1,
+        "slot": "Hand",
+        "text": "Fast. Uses (2 secrets).\n[action] Spend 1 secret: <b>Investigate.</b> Either investigate with a base [intellect] skill of 5, or get +2 [intellect] for this investigation. If you succeed, you discover 1 additional clue at your location.",
+        "traits": "Item. Tome.",
+        "type_code": "asset",
+        "xp": null
+    },
+    {
+        "code": "08736",
+        "cost": 1,
+        "deck_limit": 1,
+        "encounter_code": "memorials_of_the_lost",
+        "encounter_position": 7,
+        "faction_code": "neutral",
+        "illustrator": "Mauro Dal Bo",
+        "name": "Sinha's Medical Kit",
+        "pack_code": "eoec",
+        "position": 236,
+        "quantity": 1,
+        "skill_wild": 1,
+        "skill_willpower": 1,
+        "text": "Uses (3 supplies).\n[fast] Exhaust Sinha's Medical Kit and spend 1 supply: Heal 1 damage or 1 horror from an investigator or [[Ally]] asset at your location.",
+        "traits": "Item. Science.",
+        "type_code": "asset",
+        "xp": null
+    },
+    {
+        "code": "08737",
+        "cost": 0,
+        "deck_limit": 1,
+        "encounter_code": "memorials_of_the_lost",
+        "encounter_position": 8,
+        "faction_code": "neutral",
+        "flavor": "Takada left these spare supplies behind. Maybe you can make use of them.",
+        "illustrator": "Vincent Dutrait",
+        "name": "Takada's Cache",
+        "pack_code": "eoec",
+        "position": 237,
+        "quantity": 1,
+        "skill_wild": 1,
+        "text": "Fast. Play during any [fast] window.]\nGain 3 resources and draw 1 card. Remove Takada's Cache from the game.",
+        "traits": "Supply.",
+        "type_code": "event",
+        "xp": null
+    },
+    {
+        "code": "08738",
+        "cost": 3,
+        "deck_limit": 1,
+        "encounter_code": "memorials_of_the_lost",
+        "encounter_position": 9,
+        "faction_code": "neutral",
+        "flavor": "\"Nobody knows these machines better than I do.\"",
```

Comment:
Delete this flavor (I think it's Takada's, but funny to think Anyu said this)

Reply:
lol, clever girl